generator/mavgen.py

```diff
@@ -180,7 +177,7 @@ def mavgen_python_dialect(dialect, wire_protocol):
         py = os.path.join(dialects, 'v20', dialect + '.py')
         xml = os.path.join(dialects, 'v20', dialect + '.xml')
         if not os.path.exists(xml):
-            xml = os.path.join(mdef, 'v1.0', dialect + '.xml')
+            xml = os.path.join(mdef, 'v1.0', dialect + '.xml')  # is v1.0 correct here??
```

Comment:
not-so-nice a change :-)

I'm guessing this might have snuck in accidentally.  I'm not 100% sure, but I think v1.0 here is just an artifact of us using v1.0 message definitions (https://github.com/ArduPilot/pymavlink/blob/master/setup.py#L40).

This needs either a comment along the same lines as that one I just referenced, or correction.  Whichever it is, it doesn't belong in this PR :-)  Could you split this patch out, please?